dockerfiles/Dockerfile.arm32v7

```diff
@@ -13,21 +31,26 @@ RUN ./configure --system-curl
 RUN make
 RUN sudo make install
 
+# Set up build args
+ARG BUILDTYPE=MinSizeRel
+ARG BUILDARGS="--config ${BUILDTYPE} --arm"
+
 # Prepare onnxruntime Repo
+WORKDIR /code
 RUN git clone --recursive https://github.com/Microsoft/onnxruntime
 
-WORKDIR /code/onnxruntime
-ARG BUILDTYPE=Debug
-
 # Start the basic build
-RUN ./build.sh --config ${BUILDTYPE}
+WORKDIR /code/onnxruntime
```

Comment:
Why do you build it multiple times?

Reply:
The three commands build 3 different phases. The first is "--update --build" and is the *real* build, which takes a long time. After that, it uses the output of the above to "--build_shared_lib" and then to build the python stuff. The second and third phases are fairly quick, since they use the output of the first "--build".

The advantage to doing it this way, in a Dockerfile especially, is that you get checkpoints in between... if the Python bindings fail, for instance, you don't need to rebuild everything before that. Fixing it and running docker build will skip everything up to the failed command and resume.